php/EE/Migration/Executor.php

```diff
@@ -103,13 +101,18 @@ private static function get_migrations_to_execute() {
     }
 
     private static function get_migrations_from_db() {
-        return Migration::all();
+        return array_column( Migration::all(), 'migration' );
     }
 
     private static function get_migrations_from_fs() {
         // array_slice is used to remove . and .. returned by scandir()
-        $migrations = array_slice( scandir( self::MIGRATION_PATH ), 2 );
-        array_walk( $migrations, function( &$migration, $index ) {
+		$migrations = scandir( self::MIGRATION_PATH );
+
+		if( ! Utils\inside_phar() ) {
+			$migrations = array_slice( scandir( self::MIGRATION_PATH ), 2 );
```

Comment:
Any reason of having the same `scandir` call when it is already there above: https://github.com/EasyEngine/easyengine/blob/2ce1a036e76540623ac39a4db091f4a422a27f10/php/EE/Migration/Executor.php#L109

Reply:
Nope. Fixed.